scripts/docker_build.sh

```diff
@@ -27,8 +27,8 @@ build_image() {
     BASE_IMAGE=${1}
     IMAGE_TAG=$(echo ${BASE_IMAGE} | tr -s :/ - ) # replace colons and slashes with hyphens
 
-    # pull the rust version out of ../rust-toolchain to keep things synched
-    RUST_TOOLCHAIN_FILE="$SCRIPT_DIR/../rust-toolchain"
+    # pull the rust version out of ../rust-toolchain.toml to keep things synched
+    RUST_TOOLCHAIN_FILE="$SCRIPT_DIR/../rust-toolchain.toml"
     RUST_VER=$(cat $RUST_TOOLCHAIN_FILE | tr -d '\n')
```

Comment:
We now need a real parser here, sadly.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/520/commits/f8793f0ad885c207bb6be24d3f7bd50bffb7beda.

I'm using the refactored out `query_toml.py` now.